.evergreen/run-deployed-lambda-aws-tests.sh

```diff
@@ -1,5 +1,4 @@
 #!/bin/bash
-set -o errexit  # Exit the script with error if any of the commands fail
```

Comment:
Are there alternative ways we can ensure that the function and cluster get cleaned up? In python we use `trap "cleanup;commands;" EXIT HUP` to ensure some commands run whenever the shell exits.

Reply:
I like that idea. Added it here: https://github.com/mongodb-labs/drivers-evergreen-tools/pull/296/commits/1c3150206b41eccb3ab79aadb2580b8237b30565